src/deploy/table_account_event_size_pref.sql

```diff
@@ -0,0 +1,19 @@
+-- Deploy maevsi:table_account_private to pg
+-- requires: schema_public
+-- requires: table_account_public
+-- requires: enum_event_size
+
+BEGIN;
+
+CREATE TABLE maevsi.account_event_size_pref (
```

Comment:
Let's continue to not use abbreviations and follow the "general to specific" order:
1. **account**: the foundation we're extending
2. **preference**: the foundation's property
3. **event**: the property's specification
4. **size**: the specification's field

```suggestion
CREATE TABLE maevsi.account_preference_event_size (
```

This makes it easy to find all api endpoints of interest on postgraphiql due to the ordering as in this mockup:

![image](https://github.com/user-attachments/assets/726eee1e-e357-44d4-b7d4-f808941f1cf2)

You can see all available account preference at a glance.

Reply:
done